examples/quasiparticle/src/QuasiparticleSteppingAction.cc

```diff
@@ -22,63 +22,47 @@
 
 //....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo....
 //Default constructor
-QuasiparticleSteppingAction::QuasiparticleSteppingAction()
-{
+QuasiparticleSteppingAction::QuasiparticleSteppingAction() {
   //Upon construction of this class, create a txt file with step information 
   fOutputFile.open("StepInformationFile.txt",std::ios::trunc);
-
-  
-  
 }
 
-//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo....
-QuasiparticleSteppingAction::~QuasiparticleSteppingAction()
-{
+QuasiparticleSteppingAction::~QuasiparticleSteppingAction() {
   fOutputFile.close();
 }
 
-//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo....
 //Alternative constructor
-void QuasiparticleSteppingAction::UserSteppingAction( const G4Step * step )
-{
+void QuasiparticleSteppingAction::UserSteppingAction( const G4Step * step ) {
   //For now, simple: look at the pre-step point volume name and the track name
-  //  std::cout << "REL stepping. PreSP volume name: " << step->GetPreStepPoint()->GetPhysicalVolume()->GetName() << ", track particle type: " << step->GetTrack()->GetParticleDefinition()->GetParticleName() << std::endl;
 
   //First up: do generic exporting of step information (no cuts made here)
   ExportStepInformation(step);
 
   clock_t timestamp;
   timestamp = clock();
-  //G4cout << "-----------------------> Time in user stepping action: " << double(timestamp)/double(CLOCKS_PER_SEC) << " seconds" << G4endl;
-  
   return;
 }
 
-
-
-//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo....
 // Do a set of queries of information to test for anharmonic decay
-void QuasiparticleSteppingAction::ExportStepInformation( const G4Step * step )
-{
+void QuasiparticleSteppingAction::ExportStepInformation( const G4Step * step ) {
```

Comment:
Extra spaces at parens and pointer.  I think you can find all occurrences with something like: `egrep '\( +'`, `egrep ' +\)'`, `grep ' \*'`.  The latter will give you false positives for math, so you don't want to use it directly with `sed`.